pkgs/test/pubspec.yaml

```
@@ -29,7 +29,7 @@ dependencies:
   shelf: ^1.0.0
   shelf_packages_handler: ^3.0.0
   shelf_static: ^1.0.0
-  shelf_web_socket: '>=1.0.0 <3.0.0'
+  shelf_web_socket: '>=1.0.0 <4.0.0'
```

All we need to do for this one is just revert this back to 3.x, and update the changelog. Then we can land/publish and unblock the shelf_web_socket change.
 
```suggestion
  shelf_web_socket: '>=1.0.0 <3.0.0'
```